docs/Advanced-Usage/Chip-Communication.rst

```diff
@@ -0,0 +1,147 @@
+.. _chip-communication:
+
+Communicating with the Chip/DUT
```

Comment:
I think we mostly need to differentiate here between Protocols, Software API, and physical transport mediums/implementations